_data/sidebar_doc.yml

```diff
@@ -495,8 +495,11 @@ entries:
     - title: Releases
       items:
 
-        - title: v1.0-rc.2
-          url: /v1.0-rc.2.html
+        - title: 1.0
+          url: /v1.0.html
+
+        - title: Known Limitations
```

Comment:
Do you think it's enough to just link from the release notes? We don't need to show the known limitations page somewhere in the sidenav?